crates/ruff_linter/src/fix/edits.rs

```diff
@@ -144,6 +146,53 @@ pub(crate) fn make_redundant_alias<'a>(
         .collect()
 }
 
+/// Fix to add the specified imports to the `__all__` export list.
+pub(crate) fn add_to_dunder_all<'a>(
+    names: impl Iterator<Item = &'a str>,
+    expr: &Expr,
+    stylist: &Stylist,
+) -> Vec<Edit> {
+    let (insertion_point, export_prefix_length) = match expr {
+        ast::Expr::List(ast::ExprList { elts, range, .. }) => (
+            elts.last()
+                .map_or(range.end() - "]".text_len(), |elt| elt.range().end()),
+            elts.len(),
+        ),
+        ast::Expr::Tuple(tup) if tup.parenthesized => (
+            tup.elts
+                .last()
+                .map_or(tup.range.end() - ")".text_len(), |elt| elt.range().end()),
```

Comment:
I believe `tup.range.end()` can just be `tup.end()`, and `elt.range().end()` can just be `elt.end()`.

Reply:
Thanks